client/components/contributepage/InfoSection.js

```diff
@@ -104,7 +110,10 @@ const InfoSection = () => {
     <Article>
       <BigList>
         <BigItem>
-          <ItemTitle>Choose an {issuesLink}</ItemTitle>
+          <ItemTitle>
+            {/* eslint-disable-next-line */}
```

Comment:
Didn't even know that was a setting - bravo 👍 
We should probably investigate if we should fix the line itself in the future. Normally there's better ways to write it, if eslint complains about it 🙂

Reply:
@Soreg I believe Lily got this in before our linting was worked out, so as soon as I opened the file it complained and I could not save it back the way she had it. This was the only way I could get the space to remain between the text and the link.
Like you said, there must be a better way 😅